marklogic-sesame/src/test/java/com/marklogic/semantics/sesame/MarkLogicRepositoryCacheTest.java

```diff
@@ -116,6 +116,7 @@ public void testSizeCommitWithWriteCache()
     {
         conn.setIsolationLevel(IsolationLevels.SNAPSHOT);
 
+        conn.clear();
```

Comment:
I had some issue with old triples beeing leftover from a previous test thats why the line is there. But would be better to put it in the @before or @after  of the unit test